tests/test_fin_evaluator.py

```diff
@@ -46,6 +49,13 @@ def test_fin_evaluator():
   v = Value('i', 0)
   e = Value('i', 0)
```

Comment:
If possible, renaming "v" and "e" to a more descriptive variable names would make it easier for traceability and future edits :)